source/Pacioli/Pacioli.Lib/Models/JournalEntry.cs

```diff
@@ -29,7 +29,7 @@ public record JournalEntry
             EnsureArgNotEmpty(debits, nameof(debits), "Debits should not be empty");
             EnsureArgNotEmpty(credits, nameof(credits), "Credits should not be empty");
 
-            bool hasCommonAccount = debits.Any(dr => credits.Any(cr => dr.Account.Equals(cr.Account)));
+            bool hasCommonAccount = debits.Any(dr => credits.Any(cr => dr.Account.Name.Equals(cr.Account.Name)));
```

Comment:
Why is only one property being compared now?

Reply:
I found the difference. I set both test data as credit and debit, where it should have been debit and debit or credit and credit.